gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -67,6 +76,11 @@ def _get_variant_merge_strategy(known_args):
         known_args.info_keys_to_move_to_calls_regex,
         known_args.copy_quality_to_calls,
         known_args.copy_filter_to_calls)
+  elif known_args.variant_merge_strategy == 'MERGE_WITH_NON_VARIANTS':
```

Comment:
Please add this to _validate_args as well.

Reply:
Done.